packages/apps/tools/src/pages/_app.tsx

```diff
@@ -10,6 +11,8 @@ import type { AppProps } from 'next/app';
 import type { FC } from 'react';
 import React from 'react';
 
+KodeMono();
```

Comment:
I think you can remove this now

Reply:
This in the Tools app, not react-ui, so we still need this 😁